script/Ciel/Day 8/08_01_CIEL08_4.txt

```diff
@@ -16,7 +16,7 @@ At night, in a park where everyone had already left.
 [sha:ef6bed8a433c056a9ca818e0bec8c7300a6a226e]{
 -- Page 10010, Offset 19093.
 -- 誰に宛てたものか<定|さだ>かではない独白が闇に吸い込まれるように消えていった。
-My vague monologue, intended for no one in particular, faded away, like sucked up by the darkness.
+My vague monologue, intended for no one in particular, fades away as if absorbed by the darkness.
```

Comment:
Since darkness doesn't _actually_ absorb or swallow stuff, there's no need for the "as if". It's already figurative language.
> fades away, swallowed by the darkness.